Package.swift

```diff
@@ -0,0 +1,33 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Kommunicate-iOS-SDK",
+    defaultLocalization: "en",
+    platforms: [.iOS(.v10)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Kommunicate-iOS-SDK",
```

Comment:
Change the name to Kommunicate and also targets to Kommunicate  as we are adding SPM support to Kommunicate SDK, not the Sample project.